k8s/environments/sbox/common-overlay/pip/azure-identity-patch.yaml

```diff
@@ -0,0 +1,6 @@
+- op: replace
+  path: /spec/resourceID
+  value: /subscriptions/a8140a9e-f1b0-481f-a4de-0[…].ManagedIdentity/userAssignedIdentities/pip-sbox-mi
```

Comment:
this is not the full resource id, you can copy it from the properties page of the identity